### DISCUSSION TITLE
Min version of Yup supported?

### DISCUSSION THREAD

**Discussion by Monte47** [react-hook-form/react-hook-form#10345] [General] [answered]
I'm trying to migrate from Formik to React Hook Form, and mostly use Yup as a schema validation. Currently on Yup version .30.0.

I'm wondering what the minimum supported version of Yup is where I can use @hookform/resolvers. I'm trying to migrate a large codebase so don't want to jump to the latest version to avoid as many breaking changes as possible for the moment.

**Comment by writes-of-spring** [comment, marked as answer]
hookform/resolvers 3.0.0 introduced a yup v 1 requirement.

I'm using yup 0.29.3 and hookform/resolvers 2.9.11 on a project without issue.

The migration will be worth it!
